src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -0,0 +1,284 @@
+    configureSpark("Drive set pid wrap max", () -> driveController.setPositionPIDWrappingMaxInput(2 * Math.PI));
+    configureSpark("Drive set pid wrap", () -> driveController.setPositionPIDWrappingEnabled(true));
+    configureSpark("Drive voltage comp", () -> driveMotor.enableVoltageCompensation(Config.Swerve.voltageComp));
+    configureSpark("Drive set position", () -> driveEncoder.setPosition(0.0));
+  }
+
+  /**
+   * Save the configurations from flash to EEPROM.
+   */
+  private void burnFlash() {
+    try {
+      Thread.sleep(200);
+    } 
+    catch (Exception e) {}
+
+    driveMotor.burnFlash();
+    angleMotor.burnFlash();
+  }
+
+  /*
+   * Sets Speed
+   * 
+   * @param desiredState
+   * @param isOpenLoop Op
+   */
+  private void setSpeed(SwerveModuleState desiredState, boolean isOpenLoop) {
+    if (isOpenLoop) {
+      double percentOutput = desiredState.speedMetersPerSecond / Config.Swerve.maxSpeed;
+      driveMotor.set(percentOutput);
+    } else {
+      errSpark("Drive set FF", 
+        driveController.setReference(
+          desiredState.speedMetersPerSecond,
+          ControlType.kVelocity,
+          0,
+          feedforward.calculate(desiredState.speedMetersPerSecond)));
+    }
+  }
+
+  /**
+   * Sets Angle
+   * 
+   * @param desiredState 
+   */
+  private void setAngle(SwerveModuleState desiredState, boolean isDisableAntiJitter) {
+    // Prevent rotating module if speed is less then 1%. Prevents jittering.
+    Rotation2d angle;
+    if (isDisableAntiJitter) {
+      angle = desiredState.angle;
+
+    // Anti jitter check
+    } else if (Math.abs(desiredState.speedMetersPerSecond) <= (Config.Swerve.maxSpeed * 0.01)) {
+      angle = lastAngle;
+
+    } else {
+      angle = desiredState.angle;
+    }
+
+    errSpark("Angle set reference", angleController.setReference(angle.getRadians(), CANSparkBase.ControlType.kPosition));
+    lastAngle = angle;
+  }
+
+  /**
+   * Returns Angle
+   * 
+   * @return Angle
+   */
+  private Rotation2d getAngle() {
+    return (new Rotation2d(integratedAngleEncoder.getPosition()));
+  }
+
+  public Rotation2d getCanCoder() {
+    return Rotation2d.fromDegrees(angleEncoder.getAbsolutePosition());
+  }
+
+  public SwerveModuleState getState() {
+    return new SwerveModuleState(driveEncoder.getVelocity(), getAngle());
+  }
+
+  public SwerveModulePosition getPosition() {
+    return new SwerveModulePosition(driveEncoder.getPosition(), getAngle());
+  }
+
+  public void periodic() {
+    //update network tables
+    currentSpeedEntry.accept(driveEncoder.getVelocity());
+    currentAngleEntry.accept(getAngle().getRadians());
+    updateFeedforward.checkForUpdates();
+    canCoderAngleEntry.accept(getCanCoder().getDegrees());
+
+    if (Config.swerveTuning) {  
+      angleOffset = Rotation2d.fromDegrees(entryAngleOffset.get());
+    }
+  }
+
+  public void queueSynchronizeEncoders() {
+    if (driveEncoder != null) {
```

Comment:
Just remove this line, since we always have angle encoder (shouldn't be checking driveEncoder anyway).